src/core/qgsmaplayer.cpp

```diff
@@ -1166,7 +1167,7 @@ bool QgsMapLayer::importNamedStyle( QDomDocument &myDocument, QString &myErrorMe
   }
 #endif
 
-  return readSymbology( myRoot, myErrorMessage );
+  return readSymbology( myRoot, myErrorMessage, QgsPathResolver() ); // TODO: support relative paths?
```

Comment:
@dakcarto I am not sure if I understand how did you mean your comments. We already have support for relative paths to SVG symbol within project directory... (just without a special ```symbols``` subdir - but of course users could decide to use such convention if they wanted)

That TODO (when implemented) is meant to add support for relative paths for .qml files - until now it was supposed that relative paths to SVG symbols were relative to a system SVG directory or the current project directory - which is wrong - and instead the directory of the .qml file should be used.